lib/postmark/error.rb

```diff
@@ -30,6 +30,7 @@ def self.build(status_code, body)
     def initialize(status_code = 500, body = '', parsed_body = {})
       self.parsed_body = parsed_body
       self.status_code = status_code.to_i
+      self.body = body
```

Comment:
Maybe we could completely remove this property - no one seems to have noticed that we don't set it at all.

Reply:
agreed

Reply:
I was thinking about this one and decided to leave it in. I do agree that it was not reported, but I think it's still good to have it. Main reasoning for that is the rare case when there is a HTTP error and response for some reason might not be JSON from http client. It would be good to have this for debugging.